tc_core/web.py

```diff
@@ -1,47 +0,0 @@
-# -*- coding: utf-8 -*-
-
-# Copyright (c) 2015, thumbor-community
-# Use of this source code is governed by the MIT license that can be
-# found in the LICENSE file.
-
-import re
-
-from thumbor.url import Url
-import tornado.web
-
-
-class RequestParser(object):
-
```

Comment:
This class should not be deleted. Used in https://github.com/thumbor-community/shortener/blob/master/tc_shortener/handlers/shortener.py#L17